src/PDS4_VGR_IngestLDD.xml

```diff
@@ -262,11 +267,10 @@
     <local_identifier>fds_scet_epoch_4</local_identifier>
     <nillable_flag>false</nillable_flag>
     <submitter_name>Matthew Tiscareno</submitter_name>
-    <definition>The Voyager spacecraft clock, known as the Flight Data System (FDS), was reset to zero multiple times, dividing each mission into FDS partitions, which count starting at 1.  If there is uncertainty regarding the correct FDS partition (that is, the correct value of spacecraft_clock_count_partition), then this attribute indicates what spacecraft_clock_start_count would be if the partition were 4.<!--Verify that start_date_time is the right thing to say here.--></definition>
+    <definition>The Voyager spacecraft clock, known as the Flight Data System (FDS), was reset to zero multiple times, dividing each mission into FDS partitions, which count starting at 1.  If there is uncertainty regarding the correct FDS partition (that is, the correct value of spacecraft_clock_count_partition), then this attribute indicates what spacecraft_clock_start_count would be if the partition were 4.<!--Verify that start_date_time is the right thing to say here.--></definition><!--There should be a schematron rule ensuring that at least two of the fds_scet_epoch_N values are used whenever spacecraft_clock_count_partition is zero.-->
     <DD_Value_Domain>
       <enumeration_flag>false</enumeration_flag>
-      <value_data_type>ASCII_Short_String_Collapsed</value_data_type>
-      <pattern>YYYY-MM-DDTHH:MM:SS.SSSSSSZ</pattern>
+      <value_data_type>ASCII_Date_Time_YMD_UTC</value_data_type>

```

Comment:
Seems like it was indeed an ASCII issue. There were some extra `&ZeroWidthSpace;` characters in the data type value strings, but that's fixed now.